guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+be afraid to just jump in. You have meaningful questions to ask, or comments
+to make. You are the only one judging yourself for not
+having some perceived level of expertise.
+
+
+## What mindset should I have?
+
+Learning new things is challenging. It's important to realize that since using
+GitHub and git is entirely new to you, it's going to be challenging. You'll 
+likely start out and not entirely be sure about what the commands you are
+copy pasting are doing. There are a few things to remember here.
+
+### Inexperienced != Incapable
+
+Every current user of GitHub, or any technology for that matter, was new at 
+some point. Think of someone that you consider proficient or expert, and 
+realize that they at some point were in the same shoes as you. The
+fact that you are new to a technology says nothing about what you
+are capable of, of your innate abilities.
+
+### It's okay to ask for help
+
+What does it mean to ask for help? You might have questions, feedback, or
+a general comment, or just want to talk to someone. Asking for help can be daunting, 
+but guess what? The SourceCred community is open to it. Once you've
+sprinted over the mental hurdle that it's okay to ask, you'll very likely
+realize that engaging in the first place is a great way to learn, and quite a bit
+of fun. You can [join discord](https://sourcecred.io/discord-invite) and
+then post your question in the `#git-help` channel.
+
+
+### I will grow
+
+Some might say that you can only truly fail at something if you decide not to try.
+But now let's imagine that you try. Even if your progress feels slow, try mentally
+zooming ahead one year and think of the enormous amount you might learn. What's
+the trick to getting there? Just don't give up, and keep trying. At some point
+things that seem weird or challenging will just click. This is how learning
+tends to work.
+
+To summarize, if you have a growth mindset, and aren't afraid to ask for help,
+the experience can not only be successful in that you will learn how to use
+GitHub, you'll also have fun and connect with others. 
+
+
+## How do I contribute?
+
+Okay, let's get down to talking about GitHub! We could start by giving you high
+level concepts first, but intsead we are going to walk you through steps
+that you can follow along, and explain as we go.
+
+### 1. Create an account on GitHub
+
+This is the fun part! If you haven't yet, you get to [create an account](https://github.com/)
+which usually comes down to generating a username that you like and providing an email.
+This is exciting because when you think about it, GitHub is just another super
+geeky social network where you'll make a lot of new friends, and connect
+with friends that you already might have on their GitHub accounts.
+Once you have an account, don't forget to upload a profile picture and add
+any other metadata of interest under [settings](https://github.com/settings/profile).
+
+### 2. Install git
```

Comment:
For people who don't already know how to use GitHub, there's a low chance they would know how to use a terminal (I certainly don't). I think this section could be more useful if it went into how to contribute on GitHub specifically using the GitHub website, such as how to review documents, etc. We've recently created a document going into that, and maybe this could link to or include an official guide once it's made (there's definitely some overlap, but both will be useful IMO).
I'd also go into how a new community member would know what there is that needs working on and how to get started. @LexBirdie would know which directions to point folx in.

Reply:
Hmm, what about if we have this be the end of the document, and then link to "Your first look at the GitHub website" (what you are referring to) and "Using git from a command line" (second, what is currently here). What I can do for now is move this last chunk of content into a separate file (something like github-terminal.md ?) and then link from here, and then you can add the link to the other document when it's ready. Does that work? I do think both descriptions are useful - a new person might be interested to try a command line (I was when I first got an Ubuntu laptop early in graduate school, although I wasn't great doing things in a terminal because I came from Windows!)

Reply:
That's a good point, I definitely wouldn't want to discourage anyone from learning how to use a command line. If there's a description of what knowledge you need before diving into the next link, that would totally work!

Reply:
I think most people are familiar enough with the concept of a command line to know if they would want to read that tutorial, so I'm not sure how you would want me to change that. If you have a suggestion for what to say I'd be happy to adjust.

Reply:
To play devil's advocate, if we provide too much description about things that a person "needs to know" in order to read a guide, that's actually creating more reasons for them to think "Oh, that's beyond what I know / am capable of doing." I think it's a much better outcome to have someone click on a guide to get more information (and then possibly try things out or learn something) than put it right up front and scare them off the bat. But that's just me. :)